### DISCUSSION TITLE
Welcome to mucommander Discussions!

### DISCUSSION THREAD

**Discussion by ahadas** [mucommander/mucommander#518] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rudyment** [comment]
How can I set 24 hour format of time?
The previous version used it, but the new one forces 12 hour on me.

**Reply by ahadas** [reply]
@rudyment check the 'Time' setting in the Preferences -> General dialog, there're two options: '12-hour format' and '24-hours format'. I don't see changes that could affect that setting in v1.1.0 but this setting might have switched to its default which is '12-hour format' due to some error
